src/patternfly/components/Alert/alert-description.hbs

```diff
@@ -1 +1,6 @@
-<p>This is a description of the notification content.</p>
\ No newline at end of file
+<span class="pf-c-alert-description{{#if alert-description--modifier}} {{alert-description--modifier}}{{/if}}"
```

Comment:
I think this file was fine as it was. If you wanted to create an element for this in the Alert component, you would call it `.pf-c-alert__description` and add it to the code docs. But I don't think that's necessary. It's also more semantic as a `p` than a `span`

Reply:
@michael-coker I changed it to a span so that the link would sit on the same line as the description, similar to how it is in the pf3 examples. I also made this change so that we could alter the description wording for each alert. Jenn mentioned that it was important to provide some context around the wording choice. But let me know otherwise and I can change it back to <p>description</p>

Reply:
@christiemolloy Ah, I see. In the case of the PF4 Alerts, the link should be part of the description since it flows with the description content. So you can use a `<p>`, just nest the link inside of it so that the link text flows with the description.

Reply:
Ok cool, that works!

Reply:
Ah, and if I understand correctly, you shouldn't create a component element just so that we can have unique text in handlebars examples. In this case, you could just just construct your handlebars files differently. Similar to how you have `alert-exclamationcircle-icon.hbs`, which is just HTML (and not an element of the component), you could change the contents of `.pf-c-alert__description` to this below, and be able to support custom content in the workspace examples without having to create an element in the component.

```
<p{{#if alert-description--modifier}} class="{{alert-description--modifier}}"{{/if}}
  {{#if alert-description--attribute}}
    {{{alert-description--attribute}}}
  {{/if}}>
  {{> @partial-block}}
</p>
```

Reply:
thanks for clarifying that, I changed that file to what you suggested above